shared/actions/fs/index.js

```diff
@@ -672,6 +672,17 @@ const letResetUserBackIn = ({payload: {id, username}}: FsGen.LetResetUserBackInP
 
 const letResetUserBackInResult = () => undefined // Saga.put(FsGen.createLoadResets())
 
+const deleteFile = (state, {payload: {path}}: FsGen.DeleteFilePayload) => new Promise(() => {
+  const opID = Constants.makeUUID()
+  return RPCTypes.SimpleFSSimpleFSRemoveRpcPromise({
+    opID,
+    path: {
+      PathType: RPCTypes.simpleFSPathType.kbfs,
+      kbfs: Constants.fsPathToRpcPathString(path),
+    },
+  })
```

Comment:
I think we need a `.catch(makeRetriableErrorHandler(action)` here. Also need to update `Constants.erroredActionToMessage` to translate error for this action.